### PR TITLE
add set/get_migrate_parameter function for HumanMonitor and QMPMonitor

### PR DESCRIPTION
**Pull request by XueqiangWei**
Signed-off-by: Xueqiang Wei xuwei@redhat.com

### CODE REVIEW

**Comment by XueqiangWei** [comment]
ID 1372198

File: virttest/qemu_monitor.py [hunk 1]
```diff
@@ -2299,6 +2322,30 @@ def get_migrate_cache_size(self):
...
+    def get_migrate_parameter(self, parameter):
+        """
+        Get the value of parameter.
+
+        :param parameter: parameter which need to get.
+        """
+        parameter_info = self.query("migrate-parameters")
+        if parameter in parameter_info:
+            return parameter_info[parameter]
+        else:
```
Comment (xutian) [inline virttest/qemu_monitor.py]: return null looks better than False.

File: virttest/qemu_monitor.py [hunk 2]
```diff
@@ -1258,6 +1258,29 @@ def get_migrate_cache_size(self):
...
+        """
+        Set parameters of migrate.
+
+        :param parameter: the parameter which need to set
+        :param value: the value of parameter
+        """
+        cmd = "migrate_set_parameter"
+        self.verify_supported_cmd(cmd)
+        cmd += " %s %s" % (parameter, value)
+        return self.cmd(cmd)
```
Comment (xutian) [inline virttest/qemu_monitor.py]: how about self.cmd("%s %s %s" % (cmd, parameter, values))

Comment (XueqiangWei) [inline virttest/qemu_monitor.py]: Yes, I agree. But, I think they are all ok, just keep the coding style in the framework, like other functions

Comment (XueqiangWei) [inline virttest/qemu_monitor.py]: No, it is not good. self.cmd is a function, and its first parameter is cmd, so call it by self.cmd(cmd)

**Comment by vivianQizhu** [comment]
LGTM, Acked-by: Qianqian Zhu qizhu@redhat.com

**Comment by xutian** [comment]
tested and it works for me. thanks @XueqiangWei

**Pull request merged**